packages/modules/qdrant/src/qdrant-container.test.ts

```diff
@@ -8,7 +8,7 @@ describe("QdrantContainer", () => {
 
   // connectQdrantSimple {
   it("should connect to the client", async () => {
-    const container = await new QdrantContainer().start();
+    const container = await new QdrantContainer("qdrant/qdrant:latest").start();
```

Comment:
This is displayed at
https://node.testcontainers.org/modules/qdrant/.

Reply:
We should pin the version. If the container implementation uses `v1.13.4`, then we should use that. Otherwise these tests will break in the future when a breaking change is released.

Reply:
Done with https://github.com/testcontainers/testcontainers-node/pull/917/commits/f13cc864d62a2f497032f2e4576e99d818ff4144.